src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -661,6 +670,12 @@ public virtual bool CanDetach()
             return true;
         }
 
+        public virtual async Task<string[]> GetTargetFeatures()
```

Comment:
You will want to move this to `gdb.cs`. It looks like lldb-mi does not support -list-target-features, but only [`-list-features`](https://github.com/lldb-tools/lldb-mi/blob/d69dadb35a859bc0aa4a43d0adb3fc4fe5856947/src/MICmdCmdSupportList.cpp).

You can leave it throw unimplmented for clrdbg.cs since that is not used anymore.